.github/workflows/build-test.yml

```
@@ -0,0 +1,30 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build-test:
+    name: Build and Test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

We should probably test v10 as well, as long as we're not requiring v12 anyway

```suggestion
        node-version: [10.x, 12.x, 14.x, 16.x]
```

---

Added! thanks